test/independent_set.jl

```diff
@@ -0,0 +1,121 @@
+using Test
+using ConstraintSolver
+const CS = ConstraintSolver
+
+import MathOptInterface
+const MOI = MathOptInterface
+
```

Comment:
Most of these aren't needed as they are part of runtests.jl, right?

Reply:
yes, it just allows running the file on its own with running all the tests. Given the low testing time of the package, we could remove them if you prefer

Reply:
Ah makes sense. I normally comment out the includes in `runtests.jl` for that :laughing: In general it would be nice to be able choose some special testsets one would like to test.

Reply:
To be consistent with the others I would remove them from this file for now.